javascript/main.js

```diff
@@ -20,6 +20,8 @@ function addList(event){
     edit = document.createElement('i');
     edit.setAttribute('class','editIcon far fa-edit');
 
+    edit.setAttribute("onclick","edit2(event)")
```

Comment:
remove parameter "event "
the function already take parameter

Reply:
Done